<ui version="4.0" >
 <class>quicklaunchAdd</class>
 <widget class="QWidget" name="quicklaunchAdd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Shortcut</string>
  </property>
  <property name="accessibleName" >
   <string>Add Shortcut</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblIcon1" >
     <property name="text" >
      <string>New icon:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KUrlRequester" name="urlIcon" >
     <property name="filter" >
      <string comment="KDE::DoNotExtract" >*.desktop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
